### PR TITLE
Publish Chrome images

### PR DESCRIPTION
**Pull request by ruescar**
Publish Chrome images

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,27 +1,66 @@
-version: 2
+version: 2.1
+commands:
+  checkout-test:
+    steps:
+      - run:
+          name: Checkout tests
+          command: cd .. && git clone -b ${TESTS_BRANCH} --single-branch git@github.com:twilio/selenoid-container-tests.git
```
Comment (syerrapragada) [inline .circleci/config.yml]: `cd ..` would it take the current directory beyond root directory?

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,27 +1,66 @@
...
-          name: Test image
-          command: cd ../selenoid-container-tests && mvn clean test -Dgrid.connection.url="http://localhost:4445/" -Dgrid.browser.name=${BROWSER_NAME} -Dgrid.browser.version=${BROWSER_CHANNEL}
+      - checkout-test
+      - when:
+          condition: << parameters.publish >>
+          steps:
+            - docker-login
+      - build-test-publish-image:
+          publish: << parameters.publish >>
+      - show-image-info
```
Comment (syerrapragada) [inline .circleci/config.yml:61]: Should we docker logout?

File: .circleci/config.yml [hunk 3]
```diff
@@ -1,27 +1,66 @@
...
+workflows:
+  version: 2
+  build:
+    jobs:
+      - build:
+          publish: false
+  manual-build-n-publish:
+    jobs:
+      - hold:
+          type: approval
```
Comment (syerrapragada) [inline .circleci/config.yml:71]: Could we not publish automatically? Do we need an approval here?

**Review by syerrapragada — CHANGES_REQUESTED** [review]
We could also merge the browser-bot tool in the same yaml file. I would prefer a separate repository for circle ci config. That way we can have the following workflows:

Brower-bot: 
    Continuously monitors for browsers when a new version is available, it publishes the version number to slack channel and triggers build image workflow.

Build image:
    Builds the image, tests and publishes the image to docker. It then triggers the orchestra workflow.

Orchestra-tests:
     Run tests on the latest built image.

File: .circleci/config.yml (hunk 1 above)
Comment (ruescar) [reply .circleci/config.yml]: No, it won't. Each step is run in the default working directory /home/circleci/project (where the code was checked out). Therefore the tests will be checked out one level above selenoid-images code, as needed, in /home/circleci

File: .circleci/config.yml (hunk 2 above)
Comment (ruescar) [reply .circleci/config.yml:61]: Sure. Done

File: .circleci/config.yml (hunk 3 above)
Comment (ruescar) [reply .circleci/config.yml:71]: This is only a temporary workflow for the scope of the current story so we can exercise publishing the image, but not for every commit. Eventually will have to be modified or removed when algorithm on when to build and publish a new image is implemented.

**Pull request closed without merging**